docs/getting_started/installation.rst

```diff
@@ -27,15 +17,14 @@ Installing with pip
 Stable version
 """"""""""""""
 
-Modin can be installed with ``pip`` on Linux, Windows and MacOS. 2 engines are available for those platforms:
-:doc:`Ray</developer/using_pandas_on_ray>` and :doc:`Dask</developer/using_pandas_on_dask>`
+Modin can be installed with ``pip`` on Linux, Windows and MacOS. 
 To install the most recent stable release run the following:
 
 .. code-block:: bash
 
   pip install -U modin # -U for upgrade in case you have an older version
 
-If you don't have Ray_ or Dask_ installed, you will need to install Modin with one of the targets:
+Two engines are available for running Modin: :doc:`Ray</developer/using_pandas_on_ray>` and :doc:`Dask</developer/using_pandas_on_dask>`. If you don't have Ray_ or Dask_ installed, you will need to install Modin with one of the targets:
```

Comment:
We should probably specify something about OmniSci like `... see below for OmniSci engine installation ...`.

Reply:
Incorporated Omnisci pointer